src/background.js

```diff
@@ -1,20 +1,20 @@
 async function sendCommandToTab(command, tab) {
-  // Let's have multiple methods to find the right button, because Spotify
-  // decided front-end development was more exciting with randomized CSS
-  // classes... but not for all UI elements of course. Yay, consistency!
-  function findAndClick(command) {
-    // https://github.com/mantou132/Spotify-Lyrics/issues/94
+  // Let's have multiple methods to find the right button, because Spotify
+  // decided front-end development was more exciting with randomized CSS
+  // classes... but not for all UI elements of course. Yay, consistency!
+  async function findAndClick(command) {
+    // https://github.com/mantou132/Spotify-Lyrics/issues/94
     const DENY = '.extension-lyrics-button';
     const VALUE_SET = Object.getOwnPropertyDescriptor(window.HTMLInputElement.prototype, 'value').set;
 
     function animate(e) {
       try {
         const anim = (i) => {
-          e.style.transform = (i % 2 == 1) ? 'scale(1.2)' : null;
+          e.style.transform = (i % 2 == 1) ? 'scale(0.8)' : null;
           if (i == 0) return;
           setTimeout(() => anim(i - 1), 80);
         };
-        anim(3);
+        anim(2);
```

Comment:
I've tested your code and while this looks nice on buttons, the animation on the sidebar navigation links looks janky. Please add a condition to only animate when tag == button (or pass a bool) in `clickAndAnimate`.

Reply:
Agreed.